lazy-either.js

```diff
@@ -1,46 +1,52 @@
 'use strict';
 
-const fs = require('fs');
-const path = require('path');
-
-const LazyEither = require('lazy-either');
-const R = require('ramda');
-const S = require('sanctuary');
-
-
-//    join :: String -> String -> String
-const join = R.curryN(2, path.join);
+const fs            = require('fs');
+
+const LazyEither    = require('lazy-either');
+const S             = require('sanctuary');
+
+const exit0         = require('./common/exit0');
+const exit1         = require('./common/exit1');
+const join          = require('./common/join');
+
+
+//    Array$traverse :: Applicative f => TypeRep f -> (a -> f b) -> Array a -> f (Array b)
+//
+//    We cannot use S.traverse because LazyEither does not currently support
+//    recent versions of the Fantasy Land specification. It's thus necessary
+//    to dispatch to `map` and `ap` (rather than their prefixed equivalents).
```

Comment:
> It's implementing the fantasy land spec, what are the fantasy-land prefixed methods?

Methods have required the `fantasy-land/` prefix since fantasyland/fantasy-land#146 which was released in [v1.0.0][]. The current version is [v3.2.0][], so LazyEither is a little out of date. I'll submit a pull request to prefix the methods. :)


[v1.0.0]: https://github.com/fantasyland/fantasy-land/releases/tag/v1.0.0
[v3.2.0]: https://github.com/fantasyland/fantasy-land/releases/tag/v3.2.0